### DISCUSSION TITLE
queryRaw Error Code 22007: db error: ERROR: invalid input syntax for type date

### DISCUSSION THREAD

**Discussion by rossinicolas** [prisma/prisma#9276] [Q&A]
Hi everybody, yesterday i updated my prisma modules from 2.20.1 to 3.0.1 and some queries in my API fail with error code 22007.
For example, the following sql query:

```ts
const result = await this.prisma.$queryRaw`
      SELECT COUNT(table1.*) AS cant
      FROM table1
      WHERE table1.myDate BETWEEN '${moment(today_date).format('YYYY-MM-DD')}' 
      AND '${moment(start_date).format('YYYY-MM-DD')}' 
      AND EXTRACT(dow FROM myDate) NOT IN (0,6);
    `;
```
    

with the 2.20 version there was no problems but now Raw query failed.

Could you tell me what's i'm doing wrong or how can i fix it.
Thanks in advance.

**Comment by ryands17** [comment]
No idea, could you share your schema so that I can reproduce?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
